### PR TITLE
Fix unit test due to trailing slash not supported in Spring boot 3

### PR DESCRIPTION
**Pull request by deibys-nuvalence**
## Description

JIRA: https://dol-ewf.atlassian.net/browse/DSG-1935

## Motivation and Context

This does not break functionality but it is MUST-have as it helps troubleshooting issue for engineers or support staff.

Also more context:
https://dol-ewf.atlassian.net/browse/DSG-1935?focusedCommentId=29085

## How Has This Been Tested?

See https://dol-ewf.atlassian.net/browse/DSG-1935?focusedCommentId=29086

## Types of changes

Patch

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation change (documentation only, will not require redeployment)
- [ ] Refactor or cleanup (functionality unchanged)
  Minor
- [ ] New feature (non-breaking change which adds functionality)
  Major
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have performed a self-review of my own code
- [X] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

- [X] I have confirmed all acceptance criteria defined by the story have been met.

### CODE REVIEW

File: service/src/test/java/io/nuvalence/platform/audit/service/controllers/AuditEventApiDelegateImplTest.java [hunk 1]
```diff
@@ -126,6 +134,7 @@ void getEvents_GivenInvalidTimeRange_ShouldReturnHttp400() throws Exception {
     }
 
     @Test
+    @Disabled
```
Comment (deibys-nuvalence) [inline service/src/test/java/io/nuvalence/platform/audit/service/controllers/AuditEventApiDelegateImplTest.java:137]: will create a ticket for these disabled tests. Some research has to be done

Comment (deibys-nuvalence) [reply service/src/test/java/io/nuvalence/platform/audit/service/controllers/AuditEventApiDelegateImplTest.java:137]: https://dol-ewf.atlassian.net/browse/DSG-1935

Comment (deibys-nuvalence) [reply service/src/test/java/io/nuvalence/platform/audit/service/controllers/AuditEventApiDelegateImplTest.java]: They have been enabled back

File: service/build.gradle [hunk 2]
```diff
@@ -78,7 +75,7 @@ dependencies {
 
 tasks.withType(JavaCompile) {
 	options.compilerArgs = [
-			'-Amapstruct.suppressGeneratorTimestamp=true'
+			'-Amapstruct.suppressGeneratorTimestamp=true', '-parameters'
```
Comment (deibys-nuvalence) [inline service/build.gradle:78]: This was the solution

Comment (JayStGelais) [reply service/build.gradle:78]: @deibys-nuvalence Nice job getting to the bottom of this one.

**Pull request merged**